astropy/tests/runner.py

```diff
@@ -170,7 +170,12 @@ def _generate_args(self, **kwargs):
         """
 
     _required_dependencies = ['pytest', 'pytest_remotedata', 'pytest_doctestplus', 'pytest_astropy_header']
-    _missing_dependancy_error = "Test dependencies are missing. You should install the 'pytest-astropy' package."
+    _missing_dependancy_error = (
+        "Test dependencies are missing: {module}. You should install the "
```

Comment:
Need f-string here? Maybe not... My string-fu is weak, but just in case.
```suggestion
        f"Test dependencies are missing: {module}. You should install the "
```

Reply:
Nope! I use `.format()` later on.

Reply:
Ah, okay. Sorry for the noise!